src/main/java/com/bravo/user/dao/model/User.java

```diff
@@ -45,5 +64,8 @@ public User(final UserSaveDto user){
     this.middleName = user.getMiddleName();
     this.lastName = user.getLastName();
     this.phoneNumber = user.getPhoneNumber();
+    this.email = user.getEmail();
+    this.password = user.getPassword();
+    this.role = "ADMIN".equals(user.getRole()) ? User.Role.ADMIN : null;
```

Comment:
A nullable role creates issues IMHO.  Undefined or unknown here using the role enum would make subsequent logic easier.  Can this map to the enum instead of string matching as well?